internal/provider/resource_gitlab_user.go

```diff
@@ -110,6 +110,12 @@ var _ = registerResource("gitlab_user", func() *schema.Resource {
 				Default:          "active",
 				ValidateDiagFunc: validation.ToDiagFunc(validation.StringInSlice(validUserStateValues, false)),
 			},
+			"namespace_id": {
+				Description: "The ID of the users namespace. Available since GitLab 14.10.",
```

Comment:
Should the note about admin only be here too?

Reply:
well, the entire resource can only be used by admins, thus I thought it would be redundant and only mentioned it for the data sources.